packages/lastrev-build-prefetch-content/src/buildTasks/writeRedirects.ts

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+import _ from 'lodash';
+import Contentful, { getGlobalSettings } from '@last-rev/integration-contentful';
+
+import writeFile from '../helpers/writeFile';
+import { CONTENT_DIR, REDIRECTS_FILE } from '../constants';
+import mkdirIfNotExists from '../helpers/mkDirIfNotExists';
+import { BuildTask } from '../types';
+
+type Redirect = {
+  from: string;
+  to: string;
+  status?: number;
+  force?: boolean;
+};
+
+const writeRedirectFile = async (redirects: string): Promise<void> => {
+  await writeFile(REDIRECTS_FILE, redirects);
+};
+
+const writeRedirects: BuildTask = async (buildConfig, { adapterConfig }): Promise<void> => {
+  const getSettings = buildConfig.useAdapter ? Contentful(adapterConfig).getGlobalSettings : getGlobalSettings;
+
+  await mkdirIfNotExists(CONTENT_DIR);
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const localeSettings: any = await getSettings({
```

Comment:
This doesn't seem to have anything to do specifically with locales. Maybe call this variable globalSettings, or just settings?